frontends/main/src/components/MITLogoLink/MITLogoLink.tsx

```diff
@@ -1,34 +1,71 @@
-import React, { HTMLAttributeAnchorTarget } from "react"
+import React from "react"
 import Image from "next/image"
 import Link from "next/link"
-import defaultLogo from "@/public/images/mit-logo-black.svg"
+import mitLogoBlack from "@/public/images/mit-logo-black.svg"
+import whiteLogoWhite from "@/public/images/mit-logo-white.svg"
+import learnLogo from "@/public/images/mit-learn-logo.svg"
+import { styled } from "ol-components"
 
 interface Props {
-  href?: string
+  logo: "mit_white" | "mit_black" | "learn"
   className?: string
-  logo?: string
-  alt?: string
-  target?: HTMLAttributeAnchorTarget
 }
 
-const MITLogoLink: React.FC<Props> = ({
-  href,
-  logo,
-  alt = "MIT Logo",
-  className,
-  target,
-}) => (
-  <Link
-    href={href || "/"}
-    title="Link to Homepage"
-    className={className}
-    target={target}
-    // eslint-disable-next-line react/no-unknown-property
-    appzi-screenshot-exclude="true"
-    rel="noreferrer"
-  >
-    <Image src={logo || defaultLogo} alt={alt} fill />
-  </Link>
-)
+const StyledImage = styled(Image)({
+  /**
+   * Needs display: block because otherwise the image is inline, which complicates
+   * parent container height calculations.
+   *
+   * See https://stackoverflow.com/a/11126701/2747370
+   */
+  display: "block",
```

Comment:
On `main`, the parent link is `display: flex` which causes its children to be block. *This was the only link styling on `main`*.

On `nextjs`, that styling was removed. But images are inline elements by default, which messes up their positioning a little. So this makes them block.